src/main/java/com/codecool/shop/model/Order.java

```diff
@@ -0,0 +1,42 @@
+package com.codecool.shop.model;
+
+import java.util.ArrayList;
+
+import static java.lang.Integer.parseInt;
+import static java.lang.Long.parseLong;
+
+public class Order {
+
+    private int id;
+    private String name;
+    private String email;
+    private long phoneNumber;
+    private String billingCountry;
```

Comment:
Extract address info separated class: `Address`. You use the same fields only put a prefix on it. It indicates that you need to encapsulate these information instead!

Reply:
It will be changed on this week too.